DependencyInjection/Compiler/GroupRunnersCompilerPass.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Liip\MonitorBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class GroupRunnersCompilerPass implements CompilerPassInterface
+{
+    /**
+     * @param ContainerBuilder $container
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $noRunner = false === $container->hasDefinition('liip_monitor.runner');
+        $noDefaultGroup = false === $container->hasParameter('liip_monitor.default_group');
+
+        if ($noRunner || $noDefaultGroup) {
+            return;
+        }
+
+        $definition = $container->getDefinition('liip_monitor.runner');
+        $container->removeDefinition('liip_monitor.runner');
+
+        $defaultGroup = $container->getParameter('liip_monitor.default_group');
+
+        $checkServices = $container->findTaggedServiceIds('liip_monitor.check');
+        $checkCollectionServices = $container->findTaggedServiceIds('liip_monitor.check_collection');
+
+        $groups = array($defaultGroup);
+        $groups = array_merge($groups, $this->getGroups($checkServices));
```

Comment:
I know, but I like this
```php
$groups = array($defaultGroup);
$groups = array_merge($groups, $this->getGroups($checkServices));
$groups = array_merge($groups, $this->getGroups($checkCollectionServices));
$groups = array_merge($groups, $this->getGroupsFromParameter($container));
````
more than that:
```php
$groups = array_merge(
    array($defaultGroup),
    $this->getGroups($checkServices),
    $this->getGroups($checkCollectionServices),
    $this->getGroupsFromParameter($container)
);
````

For me, it is better readable: a bit shorter (in terms of line count) and these four quite uniform statements are simpler than one large statement, containing more parts.

Ok, in the end, the difference is quite small. say, and I will change it.